src/style.css

```diff
@@ -0,0 +1,44 @@
+:root {
+  --star-size: 16px;
+  --container-size: 96px;
+}
+
+* {
+  margin: 0;
+}
+
+.container {
+  height: var(--container-size);
+  width: var(--container-size);
+}
+
+.stars {
+  display: flex;
+}
+
+.stars:last-child {
+  margin-right: 0;
+}
```

Comment:
`:last-child` is a pseudo-class that chooses the last element between the siblings, not from the parent's perspective. Also, you set `.stars__star:last-child ` later correctly.